pkg/controller/operator.go

```diff
@@ -45,7 +45,7 @@ type Operator struct {
 // 3. Initiates the recorder and starts the periodic record pruneing
 // 4. Starts the periodic gathering
 // 5. Creates the insights-client and starts uploader and reporter
-func (s *Operator) Run(ctx context.Context, controller *controllercmd.ControllerContext) error {
+func (s *Operator) Run(ctx context.Context, controller *controllercmd.ControllerContext) error { //nolint: funlen
```

Comment:
Why did you ignore the `funlen` here, but at the `status.NewController` you put it into separated lines?

Reply:
here `funlen`  is about how many statements there are in the `Run` method so the `Run` would need to be chopped into smaller functions, while `status.NewController` had a really long function signature which was easy to fix :)